vendor/bat-native-ledger/src/bat_client.cc

```diff
@@ -91,8 +96,8 @@ void BatClient::requestCredentialsCallback(
   std::vector<uint8_t> publicKey;
   std::vector<uint8_t> newSecretKey;
   braveledger_bat_helper::getPublicKeyFromSeed(secretKey,
-                                               publicKey,
-                                               newSecretKey);
+                                               &publicKey,
```

Comment:
Linter does not pick these up yet, but we should not be using camel-case in C++, i.e. `newSecretKey` and should be using snake-case, i.e. `new_secret_key`

Reply:
agree, but let's do this is a separate PR